src/sgame/sg_spawn_mover.cpp

```diff
@@ -1859,7 +1880,7 @@ void SP_func_door_rotating( gentity_t *self )
 			self->die = Think_MoverDeath;
 		}
 	}
-	else
+	else if ( not ( TOGGLE & self->mapEntity.spawnflags ) )
```

Comment:
Why do other doors need to create a "door_sensor" entity but toggling doors don't?

Reply:
Because you don't want a door that requires being toggled to automatically open when a player approaches it (as it would never close again, or would close if somebody else approached the door while in the open state).

Doors with a name/targetname will also not create the `door_sensor`.

Reply:
I guess if the door is toggled by another entity then it has to have a name? So the "else if" would never be reached anyway.

Reply:
Yes, it would need a name in that context. That being said, hypothetically, the entity could be fired by other means (entityFire command, even one summoned by lua).